src/rules/no-important.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @fileoverview Rule to disallow !important flags.
```

Comment:
```suggestion
 * @fileoverview Rule to disallow `!important` flags.
```

It would be better to wrap `!important` with backticks, since ESLint's internal rules follow the same convention.

https://github.com/eslint/eslint/blob/main/lib/rules/no-const-assign.js#L20